src/test/scala/org/hammerlab/guacamole/pileup/PileupSuite.scala

```diff
@@ -89,9 +86,6 @@ class PileupSuite extends GuacFunSuite with TableDrivenPropertyChecks {
       TestUtil.makeRead("TCGACCCTCGA", "4M3I4M", 1, "chr1", Some(Seq(10, 15, 20, 25, 5, 5, 5, 10, 15, 20, 25)))
     )
 
-    val noPileup = Pileup(reads, "chr1", 0, reference.getContig("chr1")).elements
-    noPileup.size should be(0)
-
```

Comment:
Ah ok so a `PileupElement` can't be created, so therefore a `Pileup` can't be. Got it - feel free to add or not, up to you. More tests never hurt.